t/redirects.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette('liff', &test) }
+  subject { WikiData.ids_from_pages('en', ['The Meaning of Liff', 'The Deeper Meaning of Liff']) }
```

Comment:
@tmtmtmtm I am going to handle multiple redirects in another PR as it's a separate issue (https://github.com/everypolitician/wikidata-fetcher/issues/46). (I added the change in the first commit but I've removed it in the fixup.) Also, I haven't yet added a test for a page without incoming or outgoing redirects as I'm not quite sure how to find one.